### PR TITLE
Bhn bugfix

### PR DESCRIPTION
**Pull request by BradHN1**
Issue #211 and several small bug fixes

### CODE REVIEW

File: v2.0/app/redux/redux-actions/adminActions.js [hunk 1]
```diff
@@ -72,7 +72,8 @@ export const reduxGetAllCommunityVendors = (community_id) => dispatch => {
 };
 
 export const reduxGetAllCommunityTestimonials = (community_id) => dispatch => {
-  apiCall('/testimonials.listForCommunityAdmin', { community_id }).then(response => {
```
Comment (Opoku-Agyemang) [inline v2.0/app/redux/redux-actions/adminActions.js:75]: I’ll vote for consistency over inconsistency.  Let’s not have to put the slash at the front

File: v2.0/app/utils/messenger.js [hunk 2]
```diff
@@ -27,7 +27,12 @@ export async function apiCall(
   const formData = new FormData();
   Object.keys(data).map(k => (formData.append(k, data[k])));
 
-  const response = await fetch(`${API_HOST}/v3${destinationUrl}`, {
+  // make leading '/' optional
+  if (destinationUrl.charAt(0) === '/') {
```
Comment (Opoku-Agyemang) [inline v2.0/app/utils/messenger.js:31]: If destination is null the whole site will crash.
To be safe do destinationUrl && ...

File: v2.0/app/containers/MassEnergizeSuperAdmin/Summary/NormalAdminHome.js [hunk 3]
```diff
@@ -111,7 +113,7 @@ class NormalAdminHome extends PureComponent {
                   <Typography variant="caption">
                     { n.subdomain }
                     <br />
```
Comment (Opoku-Agyemang) [inline v2.0/app/containers/MassEnergizeSuperAdmin/Summary/NormalAdminHome.js:115]: Nice !

File: v2.0/app/utils/messenger.js (hunk 2 above)
Comment (BradHN1) [reply v2.0/app/utils/messenger.js:35]: Added protection against null or 0 length destinationUrl

**Review by Opoku-Agyemang — APPROVED** [review]

**Pull request merged**